esbuild.js

```diff
@@ -0,0 +1,12 @@
+const esbuild = require('esbuild');
+
+esbuild
```

Comment:
You could also use the esbuild cli if you didn't want to maintain this code.

Reply:
The CLI is great for simple cases, but once we start adding plugins (as in #96) it becomes unwieldy